packages/server/lib/cypress.js

```diff
@@ -153,7 +153,15 @@ module.exports = {
 
     const options = argsUtils.toObject(argv)
 
+    debug('from argv %o got options %o', argv, options)
+
+    if (options.headless) {
+      // --headless is same as --headed false
```

Comment:
should probably throw here if `options.headed` is true

Reply:
added